app/assets/stylesheets/base.scss

```diff
@@ -10,14 +10,35 @@ a {
 
 h1 {
   color: $sul-h1-font-color;
-  font-size: 2.3em;
+  font-size: 2.1em;
   letter-spacing: -0.05em;
   padding-bottom: 0;
 }
 
 h2 {
   color: $sul-h2-font-color;
-  font-size: 1.6em;
+  font-size: 1.3em;
+  font-weight: 100;
+}
+
+@media (min-width: 980px) {
```

Comment:
I think these media settings are backwards. 

small media / default has this:
h1{font-size:37px;font-size:2.3125rem}  
h2{font-size:26px;font-size:1.625rem;font-weight:100;color:#8c1515}

large media has this:
@media (min-width: 48em){h1{font-size:45px;font-size:2.8125rem}}
@media (min-width: 48em){h2{font-size:31px;font-size:1.9375rem}}